datumaro/plugins/data_formats/imagenet.py

```diff
@@ -19,6 +19,7 @@
 
 class ImagenetPath:
     IMAGE_DIR_NO_LABEL = "no_label"
+    SEP_TOKEN = ":"
```

Comment:
I could not catch up the reason we need to use this. Why do we apply this kinds of changes? And is it related to LossDynamicsAnalyzer?

Reply:
As you can see in the notebook, this example requires `ImagenetWithSubsetDirsExporter`. However, the legacy code was implemented to assign the item id by
```python
item_id = osp.join(label, image_name)
```

It makes import, export, and re-import makes impossible. For example,

`Import`
```
train/label_0/img_0.jpg
```
turns into `export`
```
train/label_0/label_0/img_0.jpg
```

We cannot import `train/label_0/label_0` directory structure using `ImagenetImporter`.